superset-frontend/src/components/FaveStar.tsx

```diff
@@ -0,0 +1,88 @@
+/**
```

Comment:
nit (optional): for next time, let's use `git mv` when moving from `jsx` to `tsx` so that the file's git history stays attached to it. Now you're on the `git blame` for all the lines in the code :)